docs/registry/configure.md

```diff
@@ -25,110 +25,63 @@ modulescript: true
-
-{{< alert title="Note" color="note" >}}
-If you are using a [rented Viam rover](/appendix/try-viam/), adding modules is disabled for security purposes.
-{{< /alert >}}
-
-{{< tabs >}}
-{{% tab name="Add a modular component" %}}
-
-To add a modular [component](/components/) from the Viam registry to your machine:
-
-1. Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
-2. Click the **+** (Create) icon next to your machine part in the left-hand menu and select **Component**.
-3. Browse the list of available component types, and select the specific modular component you'd like to add.
-
-   {{<imgproc src="registry/configure/add-component-by-category.png" resize="500x" style="width: 400px" declaredimensions=true alt="The add a component modal showing results for the intel realsense module when searching by the category 'camera'">}}
-
-   You can also start typing to search for a module by name or to narrow down your search results.
-
-   {{<imgproc src="registry/configure/add-component-by-name.png" resize="500x" style="width: 400px" declaredimensions=true alt="The add a component modal showing results for the intel realsense module when searching by the name 'realsense'">}}
-
-4. After selecting the modular component, enter a name or use the suggested name for your modular component and click **Create** to add it to your machine's configuration.
+## Create your own modular resources
 
-   {{<imgproc src="registry/configure/add-component-screen.png" resize="500x" style="width: 400px" declaredimensions=true alt="The add a component modal showing the intel realsense module pane, with the 'Add module' button shown">}}
+If none of the existing modular resources in the Viam registry support your use case, you can create your own modules to provide your own modular resources:
 
-   Be sure the modular component you select supports the [platform](/cli/#using-the---platform-argument) you intend to use it with, such as `linux arm64`.
-   You can see which platforms the module supports at bottom of the module information screen before you add it.
-
-When you add a modular component from the registry, it appears on the **CONFIGURE** tab like any other component.
-
-If the component requires you to configure specific **Attributes**, navigate to the **CONFIGURE** tab and hover over the component in the machine {{< glossary_tooltip term_id="part" text="part" >}} tree in the upper left-hand corner.
-Click on the **...** menu and select **Go to homepage** to view the specific attribute requirements on the module's GitHub page.
+{{< cards >}}
+{{% card link="/how-tos/create-module/" class="fit-contain" %}}
+{{% card link="/how-tos/sensor-module/" class="fit-contain" %}}
+{{< /cards >}}
 
-To delete a modular component, navigate to the component's card on the **CONFIGURE** tab.
-Click on the **...** icon and click **Delete**.
-Confirm your selection.
+## Add a modular resource to your machine
```

Comment:
I hesitate to split module and modular resource configuration like this because in builder mode, you add both simultaneously, so this could be confusing. It's sort of like data capture vs sync.

Reply:
I'd think the screenshot will pull that together, please try it.